openfasoc/generators/common/README.md

```diff
@@ -0,0 +1,47 @@
+# Description
+This folder contains python script files that are common to all generators, used as config or result parse files for generator flows and simulations
+
```

Comment:
Adding a table of contents would be good.

Reply:
Github already adds a table of contents for READMEs with more than 2 headings I think

Reply:
It would still be good to add one since not everyone will be using the feature or accessing the README via Github.

Reply:
Yes ill rectify this soon